backend/clickhouse/migrations/000084_rewrite_trace_attributes_mv.up.sql

```diff
@@ -0,0 +1,35 @@
+DROP VIEW IF EXISTS trace_attributes_mv;
+CREATE MATERIALIZED VIEW IF NOT EXISTS trace_attributes_mv TO trace_key_values (
+    `ProjectId` Int32,
+    `Key` LowCardinality(String),
+    `Day` DateTime,
+    `Value` String,
+    `Count` UInt64
+) AS
+SELECT ProjectId,
+    arrayJoin(TraceAttributes).1 AS Key,
+    toStartOfDay(Timestamp) AS Day,
+    arrayJoin(TraceAttributes).2 AS Value,
+    count() AS Count
+FROM traces
+WHERE (
+        Key NOT IN (
+            'level',
+            'trace_state',
+            'span_name',
+            'span_kind',
+            'service_name',
+            'service_version',
+            'message',
+            'secure_session_id',
+            'span_id',
+            'trace_id',
+            'parent_span_id',
+            'duration'
```

Comment:
Is this a good time time to sync these with the columns in https://github.com/highlight/highlight/blob/main/backend/clickhouse/traces.go#L49?

Think this includes `has_errors` and `environment`, but not sure if we want to include the others:
- would we need to include `uuid`, `project_id`, `timestamp`?
- `status_code` and `status_message`? Not sure we use these
- I don't believe `level` is a column we use in traces, but we are filtering out?

Reply:
- I don't think we need to include uuid, project_id, timestamp since we don't need search suggestions for those
- not sure how `status_code` or `status_message` are being used rn - but we don't currently write those in a materialized view so it shouldn't be necessary to add them to the filter here
- agree that we don't need to filter out level, removing that